wrapper/CSharp/wolfSSL-TLS-Client/wolfSSL-TLS-Client.cs

```diff
@@ -60,14 +59,36 @@ private static int myVerify(int preverify, IntPtr x509_ctx)
         return preverify;
     }
 
+    /// <summary>
+    /// Checks if the SNI option was enabled via command line.
+    /// Must be enabled with ./configure --enable-sni when configuring
+    /// wolfSSL.
+    /// <param name="args">Parameters passed via command line</param>
+    /// </summary>
+    private static bool haveSNI(string[] args)

```

Comment:
Please have this function return `int` with the index for the `i+1` instead of `bool`, so you don't have to hard code `args[1].Trim();` below. Use -1 to indicate not found.